.env

```
@@ -1 +1,5 @@
+API=https://exam.elevateegy.com/api/v1
 NEXT_PUBLIC_API_URL=https://exam.elevateegy.com/api/v1
+NEXT_PUBLIC_Basic_API_URL="http://localhost:3000/api"
```

dotenv-linter (3.3.0)

[warning] [LowercaseKey] The NEXT_PUBLIC_Basic_API_URL key should be in uppercase

(LowercaseKey)